cli/list.go

```diff
@@ -265,21 +264,7 @@ func getPitrList(cn *pbm.PBM, size int, full bool) (ranges []pitrRange, rsRanges
 		sh[s.RS] = struct{}{}
 	}
 
-	var buf []string
 	for _, tl := range pbm.MergeTimelines(rstlines...) {
-		bcp, err := cn.GetLastBackup(&primitive.Timestamp{T: tl.End, I: 0})
-		if errors.Is(err, pbm.ErrNotFound) {
-			continue
-		}
-		if err != nil {
-			return nil, nil, errors.Wrapf(err, "get backup for timeline: %s", tl)
-		}
-		buf = buf[:0]
-		bcpMatchCluster(bcp, sh, inf.SetName, &buf)
-
-		if bcp.Status != pbm.StatusDone || !version.Compatible(version.DefaultInfo.Version, bcp.PBMVersion) {
-			continue
-		}
```

Comment:
I don't think we should unconditionally remove these checks (here and in status as well). It should be omitted only if an unchecked (unsafe or whatever) option is set.